website/docs/patterns/button-alignment/partials/overview/overview.md

```diff
@@ -0,0 +1,98 @@
+
+**Constrained** refers to a component or pattern that is wrapped within a container element, while **unconstrained** refers to components that exist at the page level.
+
+- **Constrained**
+    - Dialogs and modals
+    - Toolbars
+    - Alerts
+    - Banners
+- **Unconstrained**
+    - General page content
+    - In-page forms
+    - The "window" in a general sense
+
+This differentiation is important because the expectations from the user are different in each context. A constrained component or pattern will often be used in an interstitial or circumstantial experience; i.e. an experience that is triggered by the user or the application illiciting a quick, progressive action.
+
+Unconstrained generally refers to a more "static" experience that isn't triggered by a user action. This is often tied to a page-level function or a navigational element.
+
+**Insert some supporting images here**
+
+## Layout and reading patterns
+
+When users read content, the pattern their eyes follow generally adheres to one or more reading pattern. By identifying and familiarizing ourselves  with these patterns we can better understand how the layout within our application, pattern, or component impacts comprehension and interpretation of the content.
+
+The three most common reading patterns are:
+
+- the Gutenberg diagram
+- Z-shaped pattern
+- F-shaped pattern
+
+### Gutenberg diagram
+
+The Gutenberg diagram refers to a pattern the eyes follow when moving or scanning through **evenly distributed, homogenous content.** This pattern is most relevant in text-heavy scenarios like novels and newspapers, but is relevant in a UX context depending on the complexity of the content.
+
+![Visual example of a Gutenberg diagram](/assets/patterns/button-alignment/gutenberg-diagram.png)
+
+#### Characteristics
+
+- The users eyes progress through the content on an **axis of orientation** which moves further away from the start edge on each line or break in the content
+- Content that falls into the weak fallow area recieves less attention
+- This pattern indicates that important content and elements should be placed upon the **reading gravity path**
+- This pattern is most prevalent in content with little to no typographic hierarchy (i.e. long-form paragraphs)
+- This pattern is considered "in harmony" with natural reading direction
+
+### Z-shaped pattern
+
+A Z-shaped pattern refers to a pattern that follows the shape of the letter Z:
+
+1. The users eyes start in the top left and move horizontally to the right
+2. Upon reaching the end (or terminal) of the line, the eyes move diagonally to the next line
+3. The readers eyes end at the bottom right of the content
+
+![Visual example of a Z-shaped pattern](/assets/patterns/button-alignment/z-shaped-pattern.png)
+
+#### Characteristics
+
+- There are _technically_ no fallow areas (areas that recieve less attention or lack comprehension) as in the Gutenberg pattern, though this is not always the case in practice.
```

Comment:
Maybe this definition of "fallow" could be referenced where it's used above too if you keep the term there.